### PR TITLE
Moving testandcover script into the Makefile

### PR DESCRIPTION
**Pull request by xiam**
This PR intends to close https://github.com/getlantern/lantern/issues/2386. The testandcover script was working OK but the travis hook was not working. I just moved the script to the `Makefile`.

### CODE REVIEW

**Comment by xiam** [comment]
Hello @aranhoide.

This was failing for you on Linux, could you help me testing and merging this PR?

Thanks!

**Comment by xiam** [comment]
Actually, please wait for https://github.com/getlantern/flashlight-build/pull/178 and https://github.com/getlantern/flashlight-build/pull/177 since they're related.

File: .travis.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ install:
 script:
   # This unencryptes envvars.bash.enc into envvars.bash so that we can run the tests
   - openssl aes-256-cbc -K $encrypted_9c9444fdc7c6_key -iv $encrypted_9c9444fdc7c6_iv -in envvars.bash.enc -out envvars.bash -d
-  - ./testandcover.bash
+  - make docker-genassets test-and-cover
```
Comment (xiam) [inline .travis.yml]: The docker-genassets will not be required after we merge https://github.com/getlantern/flashlight-build/pull/178

File: .travis.yml [hunk 2]
```diff
@@ -1,10 +1,10 @@
 language: go
 
 go:
-  - 1.4.1
+  - 1.4.2
 
 install:
-  - sudo apt-get install libgtk-3-dev libappindicator3-dev
+  - sudo apt-get install make libgtk-3-dev libappindicator3-dev npm -y
```
Comment (xiam) [inline .travis.yml]: npm is only required for docker-genassets.

**Comment by aranhoide** [comment]
It works fine after a `make clean`.  It seems the offending object was sticking around from previous attempts.

**Pull request merged**